docs/source/traditional-ml/serving-multiple-models-with-pyfunc/notebooks/MME_Tutorial.ipynb

```diff
@@ -0,0 +1,863 @@
+    "        columns=[\"x1\", \"x2\", \"x3\", \"y\"],\n",
+    "        index=np.tile(rng, n_observations_per_date),\n",
+    "    )\n",
+    "    df[\"dow\"] = df.index.dayofweek\n",
+    "    return df\n",
+    "\n",
+    "\n",
+    "DF = create_weekly_dataset(n_dates=30, n_observations_per_date=500)\n",
+    "print(DF.shape)\n",
+    "DF.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "7e7fe969-003b-4e15-bb5d-0a4b8f66b710",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "#### 1.2 - Train Models for Each Day of the Week"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "eef7c545-d93e-4d96-ac8e-9ad2e6a7f305",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Successfully registered model 'DOW_model_3'.\n",
+      "Created version '1' of model 'DOW_model_3'.\n",
+      "Successfully registered model 'DOW_model_4'.\n",
+      "Created version '1' of model 'DOW_model_4'.\n",
+      "Successfully registered model 'DOW_model_5'.\n",
+      "Created version '1' of model 'DOW_model_5'.\n",
+      "Successfully registered model 'DOW_model_6'.\n",
+      "Created version '1' of model 'DOW_model_6'.\n",
+      "Successfully registered model 'DOW_model_0'.\n",
+      "Created version '1' of model 'DOW_model_0'.\n",
+      "Successfully registered model 'DOW_model_1'.\n",
+      "Created version '1' of model 'DOW_model_1'.\n",
+      "Successfully registered model 'DOW_model_2'.\n",
+      "Created version '1' of model 'DOW_model_2'.\n"
+     ]
+    }
+   ],
+   "source": [
+    "for dow in DF[\"dow\"].unique():\n",
+    "    # Create dataset corresponding to a single day of the week\n",
+    "    X = DF.loc[DF[\"dow\"] == dow]\n",
+    "    X.pop(\"dow\")  # Remove DOW as a predictor column\n",
+    "    y = X.pop(\"y\")\n",
+    "\n",
+    "    # Fit our DOW model\n",
+    "    model = RandomForestRegressor().fit(X, y)\n",
+    "\n",
+    "    # Infer signature of the model\n",
+    "    signature = infer_signature(X, model.predict(X))\n",
+    "\n",
+    "    with mlflow.start_run():\n",
+    "        model_path = f\"model_{dow}\"\n",
+    "\n",
+    "        # Log our DOW model with signature\n",
+    "        mlflow.sklearn.log_model(model, model_path, signature=signature)\n",
+    "        mlflow.set_tag(\"dow\", dow)\n",
+    "\n",
+    "        # Register our DOW model\n",
+    "        model_uri = f\"runs:/{mlflow.active_run().info.run_id}/{model_path}\"\n",
+    "        mlflow.register_model(model_uri, f\"{DOW_MODEL_NAME_PREFIX}{dow}\")"
```

Comment:
We could register the model when logging the model: mlflow.sklearn.log_model(registered_model_name=...)